### ISSUE TITLE
[Feature request] Dark theme

### ISSUE THREAD

**Issue by KaKi87** [BramVanRensbergen/Reddit-Revamped#3]
Hi,

It would be nice to have a dark theme for this userstyle.

Thanks

**Comment by BramVanRensbergen** [comment]
Hi, there is dark theme included, to enable it you'd need to enable the night mode of Reddit Enhancement Suite (RES):
![image](https://github.com/user-attachments/assets/4549f665-a7fb-424d-a710-c01b11b768fe)

It's not possible to use it without RES as it depends on how RES adapts the html tags, but if you're not familiar with [RES](https://redditenhancementsuite.com/), it's in any case a good idea to take a look even if you're not using my skin! 

If you're using RES nightmode but encounter some issues with how my skin renders it, let me know and I'll take a look

**Comment by KaKi87** [comment]
Hi,

Yes I know about RES, but its color choices are as old-looking as the original Old Reddit.
It needs *revamping*, if you will.
For example, here's another Reddit userstyle that's dark by design : https://github.com/Jorengarenar/RedditOldRedesignedDark

Thanks

**Comment by BramVanRensbergen** [comment]
If you activate night-mode while my skin is on, you're not looking at the original RES nightmode but at RedditRevamped's night mode. Like the light mode it's fully redesigned including a different color palette:
![image](https://github.com/user-attachments/assets/a34e8598-5af9-4bc4-8166-5ab208810e49)

Vs the original RES night-mode:
![image](https://github.com/user-attachments/assets/92df24ca-a8de-453d-bcf4-2fb6ea360b48)

In other words a revamped night mode is already included. It uses the RES toggle but otherwise the design is mostly unrelated.
I'm sure it's not perfect so if you have some specific suggestions I can take a look, but a full redesign isn't really in the cards at the moment I'm afraid!

**Comment by BramVanRensbergen** [comment]
Thanks for the suggestion, I'll see if I can figure out adding support for this (when the skin was created this property didn't exist yet so the RES toggle was the only dynamic option!). Could be some weeks till I get around to looking into it.

**Comment by BramVanRensbergen** [comment]
Took a look. While it would be possible to implement this on the level of the skin it is not trivial as my current dark mode is built on top of RES nightmode (i.e. would mean including the original RES nightmode css in my skin or rewriting much of my css).

However, you can configure RES to follow the OS light/dark setting, meaning it will activate nightmode when your OS or browser are set to use darkmode and deactivate it when not.

Go the RES nightmode settings: https://www.reddit.com/prefs/#res:settings/nightMode
and set automatic night mode to system preference:
![image](https://github.com/user-attachments/assets/978ee55a-fb03-4886-8014-5e7bedd75d8e)

--> with this, reddit in general, including my skin, should follow the behaviour of your OS/browser light/darkmode

(page must be reloaded for it to take effect, if not working you can try the command line command documented [here](https://www.reddit.com/r/RESissues/comments/347cgt/proper_way_of_toggling_night_mode_when_automatic/cqs0ct9/))

If I ever get around to rewriting the skin I'll adapt it to use prefers-color-scheme as I agree this makes sense. But for now configuring RES as above is the only way to adopt OS light/dark mode dynamically.
